packages/components/src/angle-picker-control/stories/index.tsx

```diff
@@ -0,0 +1,56 @@
+/**
+ * External dependencies
+ */
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { AnglePickerControl } from '..';
+
+const meta: ComponentMeta< typeof AnglePickerControl > = {
+	title: 'Components/AnglePickerControl',
+	component: AnglePickerControl,
+	argTypes: {
+		value: { control: { type: null } },
+	},
+	parameters: {
+		actions: { argTypesRegex: '^on.*' },
+		controls: {
+			expanded: true,
+		},
+		docs: { source: { state: 'open' } },
+	},
+};
+
+export default meta;
+
+const AnglePickerWithState: ComponentStory< typeof AnglePickerControl > = ( {
+	onChange,
+	...args
+} ) => {
+	const [ angle, setAngle ] = useState< number >( 0 );
```

Comment:
Notable change: the docs mentioned that both `value` and `onChange` are required props, but until this PR the Storybook example would have an initial state of `undefined`. I've fixed the issue by changing the initial value to `0`, but we should make sure that the component is used as expected across the codebase.